oauth-common/src/main/java/io/strimzi/kafka/oauth/common/OAuthAuthenticator.java

```diff
@@ -0,0 +1,106 @@
+package io.strimzi.kafka.oauth.common;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.IOException;
+import java.io.UnsupportedEncodingException;
+import java.net.URI;
+import java.net.URLEncoder;
+import java.nio.charset.StandardCharsets;
+import java.util.Base64;
+
+import static io.strimzi.kafka.oauth.common.TokenIntrospection.introspectAccessToken;
+
+public class OAuthAuthenticator {
+
+    private static final Logger log = LoggerFactory.getLogger(OAuthAuthenticator.class);
+
+    public static TokenInfo loginWithAccessToken(String token) {
+        log.warn("loginWithAccessToken() - pass-through access_token: " + token);
```

Comment:
Should we print the token like this? Shouldn't we just print the information abotu loggign with token without the actual token?

Reply:
This should be debug or trace level logging, and possibly include only first X chars of the token.